test/testUtils.ts

```diff
@@ -31,11 +31,21 @@ export async function getOptionsForFixture(file: string[]) {
 }
 
 async function getTextEditorOptions() {
+	let resolved = false;
+
 	return new Promise<TextEditorOptions>(resolve => {
 		window.onDidChangeTextEditorOptions(e => {
+			resolved = true;
 			assert.ok(e.options);
 			resolve(e.options);
 		});
+
+		setTimeout(() => {
+			if (!resolved) {
+				assert.ok(window.activeTextEditor.options);
+				resolve(window.activeTextEditor.options);
+			}
+		}, 100);

```

Comment:
This `setTimeout` block is added because `onDidChangeTextEditorOptions` is not always called. It's not called when the options being applied to the current text editor are the same as the current options, in this case, the test hang because this method does not resolve. By adding this block, it will wait for 100ms and if `onDidChangeTextEditorOptions` isn't called, it will resolve the current active texteditor options.